.github/workflows/build.yml

```diff
@@ -27,8 +27,8 @@ jobs:
       - name: Run build
         run: yarn build
 
-  # Publishing is done in a separate job to allow
-  # for all matrix builds to complete.
+  # Publishing is done in a separate job to using
```

Comment:
```suggestion
  # Publishing is done in a separate job using
```

Reply:
@Gonzalo-Avalos-Ribas thanks! have fixed it.